docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -483,6 +483,127 @@ The TransactionManager controls the timeout of database queries.
+3. Payload offloading must not be disabled after resource data has been ingested;
+4. When payload offloading is enabled, FHIR resources are limited to 5000 MiB in size when rendered as uncompressed JSON. The practical limit is likely to be lower due to memory usage and typical socket and transaction timeouts.
+
+
+#### 3.3.2.2 Pick a Container Name
+
+Each tenant/datasource combination requires its own container. Use a name that can be easily identified as belonging to the tenant and datasource, subject to the Azure Blob service naming restrictions.
+
+#### 3.3.2.3 Create the `database.properties` file
+
+Create a properties file containing the RDBMS connection information. Note that the user should not be the FHIRADMIN user which is only used by the RDBMS schema creation tool. Use the database user configured in the IBM FHIR Server `datasource.xml` file. Following the principle of least privilege access, this user typically has just the right set of privileges for the application to use objects in the FHIR data (`fhirdata`) schema:
+
+```
+db.host=localhost
+db.port=5432
+db.database=fhirdb
+user=fhirserver
+password=change-password
+currentSchema=fhirdata
+```
+
+#### 3.3.2.4 Add Payload Offload Configuration to `default/fhir-server-config.json`
+
+In the main `default/fhir-server-config.json`, configure the `fhirServer/persistence/factoryClassname` as shown below and add a `fhirServer/persistence/payload` block containing the connection information for each datasource you have defined under `fhirServer/persistence/datasources` (typically there is just one datasource called `default`). 
+```
+{
+    "__comment": "FHIR Server configuration",
+    "fhirServer": {
+        ...
+        "persistence": {
+            "factoryClassname": "com.ibm.fhir.persistence.blob.FHIRPersistenceJDBCBlobFactory",
+            "datasources": {
+                "default": {
+                    ...
+                }
+            },
+            "payload": {
+                "default": {
+                    "__comment": "Azure Blob (azurite docker) configuration for storing FHIR resource payload data",
+                    "type": "azure.blob",
+                    "connectionProperties" : {
+                        "connectionString": "your-azure-connection-string",
+                        "containerName": "default-default"
+                    }
+                }
+            }
+        }
+    }
+}
+```
+
+Container names allowed by Azure Blob are more restrictive than LinuxForHealth FHIR Server tenant and datasource names (for example they must be lower case and `_` is not allowed - see the [official documentation](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blobs-introduction#containers) for more details). For this reason, the container name for each tenant and datasource must be specified in the `fhirServer/persistence/payload/connectionProperties/containerName` property.
```

Comment:
Ah, now here is the content I was expecting to see in 3.3.2.2.
I also see the benefit of having it here since it explains why containerName is required.
Maybe just leave it as is and I'll mark my other comment as resolved too.